flink-tests/src/test/java/org/apache/flink/test/runtime/IPv6HostnamesITCase.java

```diff
@@ -146,15 +145,13 @@ private Inet6Address getLocalIPv6Address() {
 
                             // test whether Akka's netty can bind to the address
                             log.info("Testing whether Akka can use " + addr);
-                            try (NetUtils.Port port = NetUtils.getAvailablePort()) {
-                                final RpcService rpcService =
-                                        RpcSystem.load()
-                                                .localServiceBuilder(new Configuration())
-                                                .withBindAddress(addr.getHostAddress())
-                                                .withBindPort(port.getPort())
-                                                .createAndStart();
-                                rpcService.closeAsync().get();
-                            }
+                            final RpcService rpcService =
+                                    RpcSystem.load()
+                                            .remoteServiceBuilder(new Configuration(), null, "8081")
```

Comment:
the external port doesnt really matter here since we explicitly set a bind port, due to which the external port is only used for advertising.